Tailer-Platform/tailer-studio
# Activities Overview

## Related issue(s)

Closes #336 


## Description

This PR add Activities Overview section.


## What does this PR do?

- [x] Rename `data-workflows` in `data-operations` in the whole codebase
- [x] Create files for `Runs` & `Confs` listing views
- [x] Add table on `Runs` listing view
- [ ] Create new `types` filter only showing on those new views
- [ ] Add remote config


## Deploy notes

Review & merge.


## Steps to test or reproduce

1. Go to `Activities overview`
2. You should see the table with data
3. Test filters


## Impacted area(s) in Application

- `Data Operations`
